src/main/java/duke/command/AddCommand.java

```diff
@@ -0,0 +1,127 @@
+package duke.command;
+
+import java.time.LocalDateTime;
+
+import duke.components.DukeException;
+import duke.components.Storage;
+import duke.components.TaskList;
+import duke.components.Ui;
+import duke.tasks.Deadline;
+import duke.tasks.Event;
+import duke.tasks.Task;
+import duke.tasks.Todo;
```

Comment:
Well Done on listing all the classes explicitly and in alphabetical order, and in other places as well.